extract-hoogle-hackage-info/extract-hoogle-hackage-info.cabal

```diff
@@ -0,0 +1,29 @@
+cabal-version: 2.4
+name: extract-hoogle-hackage-info
+version: 0.0.0.0
+author: tbagrel1
+maintainer: thomas.bagrel@tweag.io
+
+executable extract-hoogle-hackage-info
+    main-is:          Main.hs
+    hs-source-dirs:   src
+    default-language: Haskell2010
+    build-depends:
+        base >=4.12 && <5.0,
+        filepath >=1.2 && <1.5,
+        optparse-applicative >=0.14 && <0.17,
+        ormolu,
+        text >=0.2 && <1.3,
+        unordered-containers >=0.2 && <0.3,
+        hashable >=1.0 && <2.0,
+        aeson >= 1.0 && <3.0,
+        pcre2 >= 2.0 && < 3.0,
+        directory >= 1.0 && < 2.0,
+        unix >= 2.0 && < 3.0,
+        tagsoup >= 0.14 && < 0.15,
+        text-format >= 0.3 && < 0.4,
+        ghc-lib-parser >=9.2 && <9.3,
+        bytestring >= 0.10 && < 0.11
+    ghc-options: -O2 -Wall -rtsopts
+    if !impl(ghc == 8.10.*)
```

Comment:
The `extract-hoogle-hackage-info` script doesn't build yet with 9.x, because of some dependencies not yet available. So I added `extract-hoogle-hackage-info` to the 8.10.x-only part of the `default.nix` build instructions, but it seems that this line is necessary because I added the `extract-hoogle-hackage-info` subproject to the `cabal.project` file. @amesgen [wrote a comment about it](https://github.com/tweag/ormolu/pull/830#issuecomment-979296670)